docs/setup/installation.md

```diff
@@ -4,7 +4,9 @@ title: Installation
 
 The easiest way to install SourceCred is to fork the [example repo](https://github.com/sourcecred/example-instance). This repo contains scripts that will install SourceCred locally and set up an example instance of SourceCred. This instance can then be configured to include just the plugins you want and point to the data sources you care about.
 
-If you want to build SourceCred from source (for instance to get bleeding edge features) or to develop on it, see the [Development Setup instructions](https://github.com/sourcecred/sourcecred#development-setup) on GitHub.
+If you want to build SourceCred from source (for instance to get bleeding edge features) or to develop on it, see the [Development Setup instructions](#development-setup).
+
+You can also run SourceCred from a [Docker image](#using-docker)  (useful for automated setups and canonical instances). 
```

Comment:
We wont be supporting the docker image anymore since we have the NPM package now

Reply:
Think this was an artifact of my git sloppiness, but removed here anyway. Don't think it warrants a separate PR worth the same amount of Cred as this whole page :)